blueprints/utils/resources/expected-results.json

```diff
@@ -67,6 +67,11 @@
   "/javascript/frameworks/polymer/polymer-router.found",
   "/javascript/frameworks/react/route-path.found",
   "/javascript/frameworks/react/index.html/search.found",
+  "/javascript/state-changing-requests/automatic-post.found",
```

Comment:
Maybe `fetch` instead of `state-changing-requests` would be better?
WDYT @hjkeller16, @mariussteffens ?